paddle/fluid/framework/io/shell.h

```diff
@@ -17,9 +17,6 @@
 #include <fcntl.h>
 #include <sys/stat.h>
 #ifdef _WIN32
-#ifndef NOMINMAX
-#define NOMINMAX  // msvc max/min macro conflict with std::min/max
-#endif
```

Comment:
This change in the file should be removed. It was added accidentally.

Reply:
After **PR-CI-Coverage** will be done, I will remove it, thanks to that there will be only 19 files changed.